schema/home/home_page_artwork_module.js

```diff
@@ -79,21 +80,20 @@ const HomePageArtworkModule = {
     },
   },
   resolve: (root, { key, id, followed_artist_id, related_artist_id }) => {
-    let params;
+    // TODO Really not entirely sure what this `display` param is about.
```

Comment:
`display`, for the most part, comes from https://api.artsy.net/api/v1/me/modules, this endpoint returns the rail `key` and a boolean value for whether or not it should be shown on the homepage. On metaphysics, we translate this response into an array with keys and values and in some cases we override the `display` value for rails that should always be present.